tuned/daemon/daemon.py

```diff
@@ -177,14 +179,14 @@ def _notify_profile_changed(self, profile_names, result, errstr):
 			exports.send_signal(consts.SIGNAL_PROFILE_CHANGED, profile_names, result, errstr)
 		return errstr
 
-	def _full_rollback_required(self):
+	def _system_shutdown_or_reboot(self):
```

Comment:
It also returns `True` if we don't know (or more precisely if `systemd` doesn't know or is misconfigured/in the failed state, etc.), thus maybe rename the method to "_system_shutdown_or_reboot_or_unknown" or similarly to clearly indicate the condition.

Reply:
I wasn't able to come up with a function name for this I liked, so I reverted the function name to what it was originally and also reverted the logic in the fuction.